pxr/usdImaging/usdImagingGL/testenv/testUsdImagingGLAnimatedLights/lightMoving.usda

```diff
@@ -2,6 +2,7 @@
 (
     startTimeCode = 1
     endTimeCode = 3
+    upAxis = "Z"
```

Comment:
Some of the tests were failing because their stage was defaulting to Y-up when they were authored with Z-up. I've changed the affected stages metadata to Z-up